React/Base/RCTUtils.m

```diff
@@ -297,16 +297,23 @@ static void RCTUnsafeExecuteOnMainQueueOnceSync(dispatch_once_t *onceToken, disp
 }
 
 #if !TARGET_OS_OSX // TODO(macOS GH#774)
-CGFloat RCTScreenScale()
+static dispatch_once_t onceTokenScreenScale;
+static CGFloat screenScale;
+
+void RCTComputeScreenScale()
 {
-  static dispatch_once_t onceToken;
-  static CGFloat scale;
+  dispatch_once(&onceTokenScreenScale, ^{
+    screenScale = [UIScreen mainScreen].scale;
```

Comment:
Previously, we didn't. That being said, `-[NSScreen backingScaleFactor]` is a thing, so I don't see why we can't add it.

Reply:
Given we had a few bugs recently about text not rendering at the right scale factor, I think it would be worth it!

Reply:
As an addendum, there are several places where we have `RCTScreenScale` blocked out on macOS, so I feel like it would be better to tackle all of these in a separate PR.

Reply:
where is it track? we shouldn't use the same GH774 since it is pretty well scope work?

Reply:
Opened #1165.